client/src/components/inputs/Button.js

```diff
@@ -1,34 +1,31 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import React, { Component } from "react";
```

Comment:
why do you change it to double quotes

Reply:
in the prettier config pr you change it to single quotes? These changes shouldn't be in this pr

Reply:
Prettier automatically did that, however with the pr that is up #100 , that will be changed back to single quotes, because I think we can all agree, single quotes are better than double quotes.